### ISSUE TITLE
Styling the navbar

### ISSUE THREAD

**Issue by EmperorYP7** [houseofgeeks/hg#16]
### Description
Format the navbar using bootstrap classes or custom CSS and make the `active` hook to rely on current URL instead of the lifecycle of the component.

### Concerned Files
https://github.com/houseofgeeks/hg/blob/front-end/client/src/components/Navbar/Navbar.css
https://github.com/houseofgeeks/hg/blob/front-end/client/src/components/Navbar/Navbar.js

### Screenshots/testimonials

Wireframe for the navbar
![Screenshot (333)](https://user-images.githubusercontent.com/62606998/92638773-6e9fd080-f2f8-11ea-87f6-3a10a29f6652.png)

**Comment by RishavMz** [comment]
The active hook can be made to rely on current URL by extracting the part from URL after the last occurrence of '/'

**Comment by EmperorYP7** [comment]
Great that you figured out @RishavMz! Do consider creating a Pull Request to solve this issue. :wink:

**Comment by RishavMz** [comment]
Ok. I'm working on it !

**Comment by RishavMz** [comment]
There seems to be a problem with CSS, I used className "logo" for the house Of Geeks logo in navbar, but it seems that it got overridden by @tusharjain0022 's commit in the footer (maybe we both were using same className )

**Comment by ankiiitraj** [comment]
That will be treated as a good first bug ! Go ahead and create a bug after #21  PR gets merged

**Comment by ankiiitraj** [comment]
is it okay to close this against #21

**Comment by EmperorYP7** [comment]
Yes @ankiiitraj the issue is resolved now. @RishavMz you may create a new issue for that.
